dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -927,7 +928,7 @@ def disabled?(selector)
 
 def wait_for_jquery
   wait_until do
-    @browser.execute_script("return (typeof jQuery !== 'undefined');")
+    @browser.execute_script("return (typeof $ !== 'undefined');")
```

Comment:
race condition where jQuery is defined before $ is assigned, but steps.rb uses `$` everywhere not jQuery => many tests break on local chromedriver on random $ calls